src/TConfig.cpp

```diff
@@ -106,6 +107,7 @@ void TConfig::FlushToFile() {
     data["General"][StrMap.data()] = Application::Settings.MapName;
     data["General"][StrDescription.data()] = Application::Settings.ServerDesc;
     data["General"][StrResourceFolder.data()] = Application::Settings.Resource;
+    data["General"][StrPassword.data()] = Application::Settings.Password;
```

Comment:
let's do this also, I feel this would be nice
```suggestion
    data["General"][StrPassword.data()] = Application::Settings.Password;
    SetComment(data["General"][StrPassword.data()].comments(), " Sets a password on this server, which restricts people from joining. To join, a player must enter this exact password. Leave empty ("") to disable the password.");
```